presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -50,6 +50,13 @@ Constructors
     two non-empty points in the input array, an empty LineString will be returned.  The returned geometry may
     not be simple, e.g. may self-intersect or may contain duplicate vertexes depending on the input.
 
+.. function:: ST_MultiPoint(array(Point)) -> MultiPoint
+
+    Returns a MultiPoint geometry object formed from the specified points. If any points in the input array is
+    null, then raise an exception. Empty points are ignored. If input array has no non-empty point, then returns
+    null. If there is only one non-empty point, returns a Point geometry. The returned geometry may not be simple
```

Comment:
The spec says the return geometry must be `MultiPoint` regardless of the number of points. Could you make it so?